app/models/api/VatScheme.scala

```diff
@@ -54,6 +55,21 @@ case class VatScheme(id: String,
 
 object VatScheme {
 
+  val exceptionKey = "2"
+  val exemptionKey = "1"
+  val nonExceptionOrExemptionKey = "0"
+  def exceptionOrExemption(eligibilityData: EligibilitySubmissionData, returns: Returns): String = {
+    (eligibilityData.appliedForException, returns.appliedForExemption) match {
+      case (Some(true), Some(true)) =>
+        throw new InternalServerException("[SubscriptionBlockBuilder] exception/exemption data missing")
```

Comment:
Just a minor thing about the message. The def isn't in SubscriptionBlockBuilder and the data isn't missing but is invalid.